src/pybind/mgr/volumes/volume.py

```diff
@@ -0,0 +1,303 @@
+import json
+import errno
+import logging
+import orchestrator
+
+from path import SubvolumeSpec
+from subvolume import SubVolume
+
+log = logging.getLogger(__name__)
+
+class Volume(object):
+    def __init__(self, mgr, orchestrator):
```

Comment:
`orchestrator` is a python module. There is no need to keep that anywhere.

Reply:
yep -- already fixed that w/ the rebase.